readme.md

```diff
@@ -26,38 +26,55 @@ Click on the button below to view the code in a Web IDE. Feel free to edit the c
 
 TodoMVC would not be possible without a strong team of [contributors](https://github.com/tastejs/todomvc/contributors) helping push the project forward each day. In addition, we have a core project team composed of:
 
+***Warning: these people like fucking eachothers butts***
+
 #### [Addy Osmani](http://github.com/addyosmani) - Founder/Lead
 
+nigga
+
 <img align="left" width="40" height="40" src="http://www.gravatar.com/avatar/96270e4c3e5e9806cf7245475c00b275.png?s=40">
   Addy is a Software Engineer at Google who originally created TodoMVC. He oversees the project direction, maintenance and organizes the planning and development efforts of the team.
 
 #### [Sindre Sorhus](https://github.com/sindresorhus) - Lead Developer
 
+he's the developmer lead just because he likes to think hes important
```

Comment:
developmer => developer

hes => he's

And start the sentence with a capital letter and end it with a dot.